src/main/kotlin/org/opensearch/replication/util/Extensions.kt

```diff
@@ -137,10 +137,6 @@ suspend fun <Req: ActionRequest, Resp: ActionResponse> Client.suspendExecuteWith
                 throw ReplicationException(e, RestStatus.TOO_MANY_REQUESTS)
             }
         }
-        log.warn(
-            "Encountered a failure while executing in $req. Retrying in ${currentBackoff / 1000} seconds" +
```

Comment:
Instead of deleting the complete log entry, should we explore keeping it and only remove the stack trace? 
We can just print the error message maybe.

Reply:
+1, this messages comes handy when troubleshooting failures.

Reply:
removed stack trace, but tried getting exception via retryException.cause, retryException.message and retryException.toString() but all of them returned empty, hence the exception message

Reply:
`[2023-09-04T13:54:45,047][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure while executing changes. Retrying in 10 seconds.OpenSearchTimeoutException can be ignored!!
[2023-09-04T13:56:55,095][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure while executing changes. Retrying in 20 seconds.OpenSearchTimeoutException can be ignored!!
[2023-09-04T13:59:15,166][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure while executing changes. Retrying in 40 seconds.OpenSearchTimeoutException can be ignored!!
[2023-09-04T14:01:55,209][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure while executing changes. Retrying in 80 seconds.OpenSearchTimeoutException can be ignored!!
[2023-09-04T14:05:15,259][INFO ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] opensearch[followCluster-1][replication_follower][T#3] @coroutine#7: Timed out waiting for new changes. Current seqNo: 1. OpenSearchTimeoutException[1m]`

Reply:
by the way , last log print is already printing out OpenSearchTimeoutException, so its fine

Reply:
Let's keep it as `Encountered a failure(can be ignored) while getting changes:  OpenSearchTimeoutException. Retrying in ...`

Reply:
```
[2023-09-04T14:57:33,124][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure(can be ignored) while getting changes:  OpenSearchTimeoutException. Retrying in 10 seconds.
<===========--> 90% EXECUTING [5m 35s]
[2023-09-04T14:59:43,157][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure(can be ignored) while getting changes:  OpenSearchTimeoutException. Retrying in 20 seconds.
<===========--> 90% EXECUTING [5m 40s]
<===========--> 90% EXECUTING [7m 55s]
[2023-09-04T15:02:03,216][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure(can be ignored) while getting changes:  OpenSearchTimeoutException. Retrying in 40 seconds.
[2023-09-04T15:04:43,262][WARN ][o.o.r.t.s.ShardReplicationTask] [followCluster-1] [follower-01][0] Encountered a failure(can be ignored) while getting changes:  OpenSearchTimeoutException. Retrying in 80 seconds.
```